packages/core/package.json

```
@@ -98,7 +98,6 @@
         "@deriv/cfd": "^1.0.0",
         "@deriv/components": "^1.0.0",
         "@deriv/deriv-api": "^1.0.13",
-        "@deriv/deriv-charts": "1.4.0",
         "@deriv/deriv-charts-beta": "npm:@deriv/deriv-charts@^2.0.0-beta.14",
```

```suggestion
        "@deriv/deriv-charts": "2.0.0-beta.14",
```
Let's keep it as `@deriv/deriv-charts` for simplicity. The `beta` tag in the version should be sufficient.

---

fixed now , Thanks